src/sources/forus-platform/pug/tpl/printables/printable-voucher-qr_code.pug

```diff
@@ -1,10 +1,40 @@
 //- Printable class is required 
 .printable.printable-voucher-qr_code
-    .printable-title(ng-if="$ctrl.voucher.type != 'product'") Uw {{ $ctrl.voucher.fund.name }}-voucher
-    .printable-title(ng-if="$ctrl.voucher.type == 'product'") Uw {{ $ctrl.voucher.product.name }}-voucher
-    br
-    qr-code(qr-value="$ctrl.qrCodeValue" qr-type="voucher")
-    .printable-description De verloopdatum van deze voucher is {{ $ctrl.voucher.expire_at_locale }}
+    .printable-main-info
+        .printable-usage-info 
+            span(i18n="voucher_printable.{{$ctrl.voucherType}}_voucher.labels.use_for")
+            br
+            strong {{ $ctrl.voucher.expire_at_locale }}
+
+        .printable-logo
+            img(ng-src="./assets/img/implementation-logo.png")    
+    
+        qr-code(qr-value="$ctrl.qrCodeValue" qr-type="voucher")
+        .printable-title {{ $ctrl.printableTitle }}
+        .printable-description(
+            i18n="voucher_printable.{{$ctrl.voucherType}}_voucher.labels.description" 
+            i18n-vars="{ printableTitle: $ctrl.printableTitle }")
+
+    .printable-additional-info(ng-if="$ctrl.voucherType == 'product' || $ctrl.organization.website")
+        .description(i18n="voucher_printable.{{$ctrl.voucherType}}_voucher.labels.show_to")  
+        .reference {{ $ctrl.voucherType != 'product' ? $ctrl.organization.website : $ctrl.organization.name }} 
+
+    .printable-organization-info
+        .contact 
+            span(
+                i18n="voucher_printable.default.labels.contact.have_questions" 
+                i18n-vars="{ printableTitle: $ctrl.printableTitle }")
+            br
+            span(i18n="voucher_printable.{{$ctrl.voucherType}}_voucher.labels.contact_us") 
+
+        span {{ "voucher_printable.default.labels.contact.phone" | translate }}: 
+        strong {{ $ctrl.organization.phone }}  
+        br
+        span {{ "voucher_printable.default.labels.contact.email" | translate }}: 
+        strong {{ $ctrl.organization.email }}   
+
+        .general-logo
```

Comment:
_webshop-common\assets\img\me\logo-bw.svg is a forus logo with less contrast; or download the one from figma